crates/network-scanner-proto/Cargo.toml

```
@@ -1,12 +1,15 @@
 [package]
 name = "network-scanner-proto"
 version = "0.0.0"
-authors = ["Devolutions Inc. <infos@devolutions.net>","Jeremy Wall <jeremy@marzhillstudios.com>"]
+authors = [
+    "Devolutions Inc. <infos@devolutions.net>",
+    "Jeremy Wall <jeremy@marzhillstudios.com>",
+    "Jon Grimes <jonkgrimes@gmail.com>"
+]
 edition = "2021"
 publish = false
-license = "Apache-2.0"
+license = "MIT, Apache-2.0"
```

Is this enough? the newly added code are under MIT

---

I think it’s fine! But you should write it like this:

```suggestion
license = "MIT/Apache-2.0"
```